ramls/oai-pmh.raml

```diff
@@ -0,0 +1,131 @@
+#%RAML 1.0
+title: OAI-PMH API
+baseUri: https://github.com/folio-org/mod-oai-pmh
+version: v1
+protocols: [ HTTP, HTTPS ]
+
+documentation:
+  - title: OAI-PMH Business Logic API
+    content: |
+      The Open Archives Initiative Protocol for Metadata Harvesting (OAI-PMH) provides
+      an application-independent interoperability framework based on metadata harvesting.
+      This module supports the OAI-PMH as a means of exposing FOLIO metadata.
+
+types:
+  oai-pmh: !include schemas/OAI-PMH.xsd
```

Comment:
It makes sense to refer to OAI-PMH root element:
```yml
types:
  oai-pmh:
    type: !include schemas/OAI-PMH.xsd#OAI-PMH
    description: OAI-PMH root element
```

Reply:
There is only one element defined in the schema (the root one) anyway. But I agree it makes sense to be more specific in RAML.

Done.